companion/lib/Instance/ModuleStore.ts

```
@@ -244,7 +244,7 @@ export class ModuleStoreService extends EventEmitter<ModuleStoreServiceEvents> {
 									moduleType: 'connection',
 								},
 								query: {
-									'module-api-version': MODULE_BASE_VERSION,
+									'module-api-version': MODULE_BASE_VERSIONS[0], // nocommit - use the whole array here
```

ESLint

[error] Unexpected 'nocommit' comment: 'nocommit - use the whole array here'.

(no-warning-comments)